python/CRUD-examples/delete/handle_pets.py

```diff
@@ -27,8 +27,8 @@ def get_all_rows(label, data_type='people'):
 
 cur = con.cursor()
 
-statement = 'update lcs_pets set owner = :1 where owner = :2'
-cur.execute(statement, (2, 1))
+statement = 'update lcs_pets set owner = :new_owner where owner = :old_owner'
+cur.execute(statement, {'new_owner':2, 'old_owner':1})
 

```

Comment:
This should really be cur.execute(statement, new_owner=2, old_owner=1). But if you don't want to make the change it isn't a big deal. :-)